### ISSUE TITLE
mdi:light-switch with `on` and `off` variants

### ISSUE THREAD

**Issue by matthiasdebaat** [Templarian/MaterialDesign#6447]
### Before submitting a new icon contribution, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](http://dev.materialdesignicons.com/roadmap/brand-icons) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I have followed the [Material guidelines](https://material.io/design/iconography/system-icons.html) for system icons.

### Tell us about your contribution.

As a full-time UX designer for Home Assistant I use this amazing project on a daily basis. There are many icons that we use and I am very grateful for that. 

We currently use `mdi:bolt` for our switch entity, as maintainers we think that could be improved. We've discussed using `mdi:toggle-switch`, but that's similar to our real toggle component. Another option is to use `mdi:light-switch`, which is too detailed for our liking and has no `on` and `off` variant.

That is why I have designed some options based on the icons above. My personal favorite is `Option 2 variant B`. Hopefully I can contribute to this project in this way. If you have any questions, I'd love to hear them! Curious what you think about this icon design.

### Share a preview of your contribution.

You can check my [Figma](https://www.figma.com/file/qijMLT1epWL7UXVFoSTUS6/Switch-Icon?node-id=1%3A9) file for more details.

![Switch Icon](https://user-images.githubusercontent.com/20203284/152420335-d6c72f28-77df-407a-9240-48810338e23c.png)

### Upload the icon source.

[mdi_light-switch.zip](https://github.com/Templarian/MaterialDesign/files/7997815/mdi_light-switch.zip)

**Comment by ohhai-kthxbai** [comment]
@matthiasdebaat this looks great and much clearer than the current. 

May I suggest that the layout of your option 2 variant B mimics an american switch (thin rocker, up for on) which as a european (british) person is unfamiliar, even opposite to what we're used to seeing - on our traditional rocker switches the most common behaviour is that the top half is out/prominent when the switch is on.

Some background:
https://diy.stackexchange.com/questions/153028/why-are-light-switch-designs-different-between-europe-and-the-us
https://www.quora.com/Why-do-US-light-switches-work-in-the-opposite-direction-to-those-used-in-the-UK

The current sideways orientation might be the way to get around the difference in people's expectations.

I hope this is useful input!

**Comment by matthiasdebaat** [comment]
Thanks for your input! 

Here in the Netherlands we use the same rocker switch design, only uses the same behavior as in the US. Wondering what the rest of the world use as behavior. I do think it's hard to design a rocker switch icon, without using duotone and keep things clear. This is one [example](https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.istockphoto.com%2Fnl%2Fvector%2Felectric-light-switch-icon-modern-minimal-flat-design-style-gm492498118-76334929&psig=AOvVaw2km_zVsCTSVvF_d-xT6U_d&ust=1647678724722000&source=images&cd=vfe&ved=0CAsQjRxqFwoTCLj1zNqfz_YCFQAAAAAdAAAAABAJ) I found online, but it's duotone. You get something like this
![rocker](https://user-images.githubusercontent.com/20203284/158966354-e9793b74-c6c2-45e8-844e-1afed091ed30.png)

**Comment by ohhai-kthxbai** [comment]
> I do think it's hard to design a rocker switch icon, without using duotone and keep things clear.

I agree, this is a tough one - finding something that's intuitive for people from different backgrounds. I remember reading about it in this book: The Design of Everyday Things, Don Norman.

I'm guessing there's also a need for these icons to be visible at a distance on a phone/dashboard, and good for colourblind users (needs strong contrast?) and other visually impaired users. 

I will have a think/look for ideas too.

**Comment by goyney** [comment]
The original `light-switch` has been updated to the "on" state (this is our pattern in the library for base icons) and reassigned from Google to [matthiasdebaat](https://github.com/matthiasdebaat):
[![light-switch](https://materialdesignicons.com/icon/light-switch)](https://materialdesignicons.com/icon/light-switch)

I've added the "off" state and the toggle variants:
[![light-switch-off](https://materialdesignicons.com/icon/light-switch-off)](https://materialdesignicons.com/icon/light-switch-off)[![toggle-switch-variant](https://materialdesignicons.com/icon/toggle-switch-variant)](https://materialdesignicons.com/icon/toggle-switch-variant)[![toggle-switch-variant-off](https://materialdesignicons.com/icon/toggle-switch-variant-off)](https://materialdesignicons.com/icon/toggle-switch-variant-off)

These will go out in the next release, which will be very shortly.
